browse-scaler/src/cmr.js

```diff
@@ -38,18 +52,27 @@ exports.getBrowseImageFromConcept = async concept => {
   }
 };
 
+/**
+ * getGranuleLevelBrowseImage: Given a or collection id, get the first associated granule
+ * @param {String} granuleId CMR concept-id. This can be a collection _or_ granule id
```

Comment:
You should rename this parameter if it can be a collection or granule. Probably just `conceptId`.

Reply:
Oh I thought I already did that - done